### PR TITLE
Create lsulfrid-Huntsville recommendation file

### PR DESCRIPTION
**Pull request by lsulfrid**
- This is a bullet point.

- [ ] This is a task list item.

### CODE REVIEW

**Comment by meeksjt** [comment]
General Comment

File: lsulfrid-Huntsville.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Huntsville
+
+### Places to eat
+- Taco Mama
+- Grill 29
+- Pints and pixels
```
Comment (meeksjt) [inline lsulfrid-Huntsville.md:6]: I love playing Galaga :heart:

Comment (lsulfrid) [reply lsulfrid-Huntsville.md:6]: My husband likes Galaga! I am really bad at all of them

**Review by meeksjt — COMMENTED** [review]
As long as you're having fun, skill is secondary.

**Pull request merged**